docroot/profiles/custom/cgov_site/modules/custom/cgov_article/src/CgovArticleTwigExtensions.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+namespace Drupal\cgov_article;
+
+use Drupal\Component\Transliteration\PhpTransliteration;
+use Drupal\Core\Language\LanguageManagerInterface;
+use Twig\Extension\AbstractExtension;
+use Twig\TwigFilter;
+
+/**
+ * Extend Drupal's AbstractExtension class.
+ */
+class CgovArticleTwigExtensions extends AbstractExtension {
+
+
+  /**
+   * The language manager.
+   *
+   * @var \Drupal\Core\Language\LanguageManagerInterface
+   */
+  protected $languageManager;
+
+  /**
+   * Constructs \Drupal\cgov_article\CgovArticleTwigExtensions.
+   *
+   * @param \Drupal\Core\Language\LanguageManagerInterface $language_manager
+   *   The language manager.
+   */
+  public function __construct(LanguageManagerInterface $language_manager) {
+    $this->languageManager = $language_manager;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getName() {
+    return 'cgov_article.CgovArticleTwigExtensions';
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getFilters() {
+    return [
+      new TwigFilter('nci_transliterate', [$this, 'transliterate']),
+    ];
+  }
+
+  /**
+   * Accented letters in the Spanish alphabet with their equivalent replacement.
```

Comment:
Not just Spanish, and not all letters in all alphabets have an exact match. There's also no mention of characters being removed.

Perhaps, "Replaces non-English characters with their closest equivalent.  Any character entities are removed."